vassal-app/src/main/java/VASSAL/build/module/map/boardPicker/board/HexGrid.java

```diff
@@ -1062,4 +1062,10 @@ public int getSnapScale() {
   public void setSnapScale(int snapScale) {
     this.snapScale = snapScale;
   }
+
+  private static boolean isClose(double x, double y) {
+    final double bounds = 0.01;
```

Comment:
Is 0.01 a good value here?

Reply:
If there's gonna be an isClose(), I recommend it go in some kind of MathUtils and the tolerance be passed as a parameter, because different situations will want different tolerances.

Reply:
It looks like this is the only case of floating point comparison in all of the code.